sci-physics/madgraph5/madgraph5-3.5.3.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# does not escape strings properly, so no python3_12 for now
+PYTHON_COMPAT=( python3_11 )
+inherit fortran-2 python-single-r1
+
+MY_PNN="MadGraph5"
+MY_PV=$(ver_rs 1-3 '_')
+MY_PN="MG5_aMC_v"
+MY_PF=${MY_PN}${MY_PV}
+
+DESCRIPTION="MadGraph5_aMC@NLO"
+HOMEPAGE="https://launchpad.net/mg5amcnlo"
+SRC_URI="https://launchpad.net/mg5amcnlo/$(ver_cut 1).0/$(ver_cut 1-2).x/+download/${MY_PN}${PV}.tar.gz -> ${MY_PNN}-${PV}.tar.gz"
+S="${WORKDIR}/${MY_PF}"
+
+LICENSE="UoI-NCSA"
+SLOT="3"
+KEYWORDS="~amd64"
+# TODO add pineapple, herwig, syscalc, pjfrym, pineappl
+IUSE=" +hepmc +lhapdf +fastjet pythia collier thepeg" # td madanalysis5 ninja samurai golem95
```

Comment:
```suggestion
IUSE="+hepmc +lhapdf +fastjet pythia collier thepeg" # td madanalysis5 ninja samurai golem95
```

Reply:
Thanks, I'll also change `hepmc -> hepmc2` in the hope of getting it consistent across packages.